metrics/api/v1/model_handlers.go

```diff
@@ -50,7 +50,6 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Consumes("*/*").
 		Produces(restful.MIME_JSON)
 
-	// The /metrics/ endpoint returns a list of all available metrics for the Cluster entity of the model.
```

Comment:
Actually I planned to remove all comments (they are duplicated in doc, couple lines below).